### PR TITLE
Update trading.md

### PR DESCRIPTION
**Pull request by stupidnewbie**
somewhat large guide rewrite.  may need some trimming.

### CODE REVIEW

File: game-guide/trading.md [hunk 1]
```diff
@@ -1,54 +1,297 @@
...
+Every good at every port his a distance factor attached to it.  The distance factor is how many sectors to the nearest port that can complete the trade for that good.  It is shown to you on the buy or sell screen for that good.
 
-In order to become evil you must locate and visit Underground HQ and have an alignment of less than +99. You can then join the underground which sets your alignment at -150.
+Each good has a supply amount.  Tier 1 goods cap at 6000, tier 2 at 5000, and tier 3 at 4000.  Buying or selling a good reduces its supply.  When the port is out of supplies you can no longer trade that good at that port.  Profit margins also get tighter as supplies get lower.
 
-The value of a given good is based on three things;
+Goods regenerate in two ways.  First, goods regenerate passively over time.  Tier 1 goods at 150 &middot; game speed, tier 2 at 100 &middot; game speed, and tier 3 at 50 &middot; game speed.  Second, trading a lower tier good increase the supply of the next tier by 90% of the amount traded.  Tier 1 goods traded will add 90% of the amount traded to tier 2.  Same with tier 2 and 3.  Tier 3 goods traded do not regenerate anything.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Goods regenerate in two ways.  First, goods regenerate passively over time.  Tier 1 goods at 150 &times; game speed, tier 2 at 100 &times; game speed, and tier 3 at 50 &times; game speed.  Second, trading a lower tier good increases the supply of the next tier by 90% of the amount traded.  Tier 1 goods traded will add 90% of the amount traded to tier 2.  Same with tier 2 and 3.  Tier 3 goods traded do not regenerate anything.
```

File: game-guide/trading.md [hunk 2]
```diff
@@ -1,54 +1,297 @@
...
-The best trade ships are listed here in order based on those ships trading the same route. Notice that having more holds does not necessarily make a ship the best, the ships speed is also an important consideration. This list only takes into account maximizing exp defense, offense and cash are not considered.
-4636 - Trade Master 4536 - Planetary Super Freighter 4276 - Inter-Stellar Trader 3830 - Planetary Freighter 3715 - Thief
+There are a few limits in code that have been stripped out of the equations as they are.  The stock amount over stock max fraction will be between 0 and 1.  Relations are capped at 1000.  The rest variables are limited by game behavior but not explicitly hard coded.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
```
A good bit of detail, but perhaps distracts from the main points here.

File: game-guide/trading.md [hunk 3]
```diff
@@ -1,54 +1,297 @@
...
+## Profit Mechanics
 
-Two thing go into determining racial relations your personal number and the global relations that a race has with yours. The global relations between you and your race will always be 500 your personal will start at 0 and can go from -500 to +500. Every time you make a successful trade your personal relations increase. If you offer less than they will take or ask more then they will give your relations suffer. The amount that relations move per trade is based on the number of cargo holds on your ship the more holds you have the more you can move personal relations. Your relations with a race will also suffer if you kill a trader of that race or attack a port owned by that race.
+Ideal Buy Price = 0.03 &middot; (Buy Value &middot; Cargo) &middot; (Distance Factor ^ 1.3) &middot; (2 - (Stock Amount / Stock Max)) &middot; (3 - 2 &middot; (Relations / 1000))
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Ideal Buy Price = 0.03 &times; (Buy Value &times; Cargo) &times; (Distance Factor ^ 1.3) &times; (2 - (Stock Amount / Stock Max)) &times; (3 - 2 &times; (Relations / 1000))  
```
I think more people will recognize "x" as multiplication.

File: game-guide/trading.md [hunk 4]
```diff
@@ -1,54 +1,297 @@
...
+## Profit Mechanics
 
-Two thing go into determining racial relations your personal number and the global relations that a race has with yours. The global relations between you and your race will always be 500 your personal will start at 0 and can go from -500 to +500. Every time you make a successful trade your personal relations increase. If you offer less than they will take or ask more then they will give your relations suffer. The amount that relations move per trade is based on the number of cargo holds on your ship the more holds you have the more you can move personal relations. Your relations with a race will also suffer if you kill a trader of that race or attack a port owned by that race.
+Ideal Buy Price = 0.03 &middot; (Buy Value &middot; Cargo) &middot; (Distance Factor ^ 1.3) &middot; (2 - (Stock Amount / Stock Max)) &middot; (3 - 2 &middot; (Relations / 1000))  
+Ideal Sell Price = 0.08 &middot; (Sell Value &middot; Cargo) &middot; (Distance Factor ^ 1.3) &middot; (1 + (Stock Amount / Stock Max)) &middot; (1 + 2 &middot; (Relations / 1000))
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Ideal Sell Price = 0.08 &times; (Sell Value &times; Cargo) &times; (Distance Factor ^ 1.3) &times; (1 + (Stock Amount / Stock Max)) &times; (1 + 2 &times; (Relations / 1000))
```

File: game-guide/trading.md [hunk 5]
```diff
@@ -1,54 +1,297 @@
...
+It is possible, by using the equations above, to calculate the perfect value for each transaction.  It is an incredibly effective method to increase experience and profit earlier in the game.  It is also dangerously distracting to sit in a defenseless trade ship while keying in supply and relation numbers for every trade.  Experiment at your own risk.
 
-These figures (and therefore the rankings) of the ships change a bit when you take into account some more variations on route length and distance from where you park, and distance to the bank.
+## Experience Mechanics
 
-Basically, the more non trading moves you need to make each day, the better the faster ships become. Non trading moves include moving to and from fed space, or between routes, or to visit the bank. For example, if you make 20 non trading moves each day, and trade on a perfect 3* route, the PSF drops down the rankings to 4th best, behind the TM, the IST and the Thief.
+Experience Gain = (Cargo Holds / 13) &middot; Distance Factor
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Experience Gain = (Cargo Holds / 13) &times; Distance Factor
```

File: game-guide/trading.md [hunk 6]
```diff
@@ -1,54 +1,297 @@
...
 
-On a strictly trading basis, if you are making more than 16 non trading moves a day the IST is ahead of the PSF for experience earned. Of course the PSF is much more defendable, but the IST can trade anywhere in the universe quite easily, so is much less predictable.
+Moving goods with a greater distance factor is a huge part of gaining experience.  The person who moves the most goods along the longest route will usually end up with the most.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Moving goods with a greater distance factor is a huge part of gaining experience.  The person who moves the most goods along the longest route will usually end up with the most experience. **Remember**: the "distance factor" is not the distance you traveled, but the distance to the nearest port that has a complementary trade for the good you are buying or selling.
```

File: game-guide/trading.md [hunk 7]
```diff
@@ -1,54 +1,297 @@
...
 
-One final note: In order to gain Federal protection at the end of your trading session, you CANNOT have any illegal goods on board. So make sure that you have all of them sold off before heading to Fed, and make sure that you have sold off your weapons to get your attack rating down. (See Alignment for more detail on attack ratings and Federal Protection.)
+Experience is also tricky because of the bartering process.  There is an ideal price that must be guessed quite accurately in order to get full experience.  But the port will offer you a price that, if your relations are not at least 1000, will not be optimal.  Guessing the correct value is incredibly difficult due to the number of variables used in the process.  Most people opt simply to work their relations up as fast as possible.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Experience is also tricky because of the bartering process.  There is an ideal price that must be guessed quite accurately in order to get full experience.  However, if your relations are 1000 or above, then the port will give you the best price without needing to barter.  Guessing the correct value is difficult due to the number of variables used in the calculation.  Most people opt simply to work their relations up as fast as possible.
```

File: game-guide/trading.md [hunk 8]
```diff
@@ -1,54 +1,297 @@
...
+## Relations
 
-Read XDemonX's Guide to Smart Trading~~
\ No newline at end of file
+Every player has two types of relation numbers for each race.  There are personal relations which are what the members of that race think of you personally.  And there are political relations which represent what races collectively think of each other.  Your own actions directly influence personal relations.  Political relations are adjusted through council actions.
+
+Racial ruling councils are made up with the top 5 experience players from that race.  Each of the 5 members are able to vote to adjust relations with another race each day.  Each of those votes adds or subtracts 15 political relations from one other race.  
+
+The highest experience council member one with 150+ alignment is made president.  The president has the power to call or veto votes.  Votes can trigger war, which sets political relations with another race to -400, or peace, which sets political relations with another race to 300.  While the president holds more power in this regard, the majority on the council ultimately has more authority.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
The highest experience council member with 150+ alignment is made president.  The president has the power to call or veto votes.  Votes can trigger war, which sets political relations with another race to -400, or peace, which sets political relations with another race to 300.  While the president holds more power in this regard, the majority on the council ultimately has more authority.
```

File: game-guide/trading.md [hunk 9]
```diff
@@ -1,54 +1,297 @@
...
+
+The war and peace votes don't matter as much as you may think.  Any race that gets to -300 is considered to be at war.  Any race that gets to 300 is considered to be at peace.  The declaration doesn't have to happen via a vote called by the president.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
The war and peace votes don't matter as much as you may think.  Any race that gets to -300 is considered to be at war.  Any race that gets to 300 is considered to be at peace.  The declaration doesn't have to happen via a vote called by the president. Instead, it can happen slowly over time by the daily relations votes by council members. 
```

File: game-guide/trading.md [hunk 10]
```diff
@@ -1,54 +1,297 @@
...
+
+The primary way to increase your personal relations is to trade.  Every successful buy or sell action improves relations.  You get one point of personal relations for every 30 cargo you trade up to 300.  This holds true until you have 500 personal relations.  After that you only get one point per trade regardless of other factors.
+
+Some players will trade special routes or use tactics specifically to gain relations quickly and get it over with.  Trading at two adjacent ports will make more relations per turn than any longer route.  It's a fast way to make it to 500 personal relations.  Remember, after 500 each trade made will generate only 1 more point.  Traders will sometimes then switch to buying or selling one good at a time to finish their relations gain.  You lose some experience and credits by doing this but it gets you through the grind very quickly.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Some players will trade special routes or use tactics specifically to gain relations quickly.  Trading at two adjacent ports will make more relations per turn than any longer route.  It's a fast way to make it to 500 personal relations.  Remember, after 500 each trade made will generate only 1 more point.  Traders will sometimes then switch to buying or selling one good at a time to finish their relations gain.  You lose some experience and credits by doing this but it gets you through the grind very quickly.
```

File: game-guide/trading.md [hunk 11]
```diff
@@ -1,54 +1,297 @@
...
+
+Also take note that effective relations are personal plus political.  Your council may declare war but if you have 2000 personal relations it does not change what ports are open to you.  The opposite may be true if your personal relations are extremely low.  As a trader, you just stay focused on keeping your effective relations above -300 in order to keep ports open to you.  And keep grinding the ports you trade with to 1000 so you get full experience and the best profits.
+
+## Overhead Costs
+
+Federal protection costs no credits but you have to get there.  Overhead to a trader is a matter of turns.  How far do I have to go to get from safety to my route?  How far do I have to go to get back?  Do I have to go out of my way to get to the bank?
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Federal protection costs no credits but you have to spend turns to get there, which can impact your trading efficiency. How far do you have to go to get from safety to your route?  How far do you have to go to return to a Federal protection after you are done trading?  Do you have to go out of my way to get to the bank?
```

File: game-guide/trading.md [hunk 12]
```diff
@@ -1,54 +1,297 @@
...
+
+While overhead values are usually small they will influence ship selection.  If you have to spend 20 turns to trade every day or two that is two hours worth of turns at 10tph or five hours at 4tph.  In that scenario you don't just spend less time getting there.  You use the saved time to trade and turn a profit.
+
+Those of you with stronger math skills will have a nagging idea of the slower ship taking longer to build turns, making less trips, and coming out equal.  The concept is correct but it clashes with the limitations of routes.  Namely supply.  There are very few scenarios where you can show up to a route and trade 500 turns out with 500 cargo holds.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Those of you with stronger math skills will have a nagging idea of the slower ship taking longer to build turns, making fewer trips, and coming out equal.  The concept is correct but it clashes with the limitations of routes.  Namely supply.  There are very few scenarios where you can show up to a route and trade 500 turns with 500 cargo holds.
```

File: game-guide/trading.md [hunk 13]
```diff
@@ -1,54 +1,297 @@
...
+
+## Finding a Trade Route
+
+Finding a route is a matter of finding ports with good distance numbers.  It's an incredible amount of work to do it by hand so players have built applications to compute the routes for us. 
+
+Download your SMR file using the button to the lower left after you have the galaxy mapped.  Use The Lazy Trader to open the SMR file you downloaded and use the trade routes function to compute routes.  You will have to configure which goods you can trade depending on your alignment and which races you can trade with depending on your relations.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Download an SMR file using the button on the lower left panel after you have the galaxy fully explored.  Use The Lazy Trader to open the SMR file you downloaded and use the trade routes function to compute routes.  You will have to configure which goods you can trade depending on your alignment and which races you can trade with depending on your relations.
```

File: game-guide/trading.md [hunk 14]
```diff
@@ -1,54 +1,297 @@
...
+
+The Lazy Trader can be found at [the game guide page for it](https://wiki.smrealms.de/tools/lazy-trader)
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
The Lazy Trader can be found at [the game guide page for it](https://wiki.smrealms.de/tools/lazy-trader).
```

File: game-guide/trading.md [hunk 15]
```diff
@@ -1,54 +1,297 @@
...
+
+## Trade Ships
+
+The essence of a trade ship is its ability to move goods.  It's not just how many fit in the hold at once.  It's how many can be transported.  This value can be referred to as *trade power*.  It can be measured by multiplying the number of cargo holds by the turn rate.  Calculate for many vessels to produce a chart like this:
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
The efficiency of a trade ship depends on both its size (number of cargo holds) and its speed. For example: A slow ship with lots of cargo holds may be good, but a ship with fewer cargo holds may be better if it is much faster. This can be referred to as *trade power*.  It is measured by multiplying the number of cargo holds by the turn rate.  Calculate for many vessels to produce a chart like this:
```

**Review by hemberger — CHANGES_REQUESTED** [review]
Great write-up! This will be very helpful for new players. I have a bunch of small suggestions for changes, which can be applied directly in the GitHub page. I'm only about 2/3 done reviewing it though, but will need to finish up tomorrow.

**Comment by MichaelKunze** [comment]
Whow! Amazing work! Thank you!

File: game-guide/trading.md [hunk 16]
```diff
@@ -1,54 +1,296 @@
...
+|Leviathan|240|7|1680|122251|668|800|0|25|100|Y|N|N|N|N|
+|Expediter|180|9|1680|1272250|785|850|0|0|0|Y|N|N|N|N|
+|Merchant Vessel|170|9|1530|825408|539|620|15|5|5|Y|N|N|N|N|
+|Drudge|250|6|1500|875683|584|955|10|0|10|Y|N|Y|N|N|
+
+This chart contains all of the more efficient ships along any non Alskant upgrade path.  Regardless of your starting ship the most effective mid tier trader is usually a Merchant Vessel.  The most effective top tier ship will depend highly upon you and the route you mean to trade.
+
+The best mid tier for half the races is a Merchant Vessel.  The few others listed here are also cost effective.  Thevians should be mindful that an Expediter cannot mount any weapons to shoot forces with.
+
+Dangerous routes are best handled by a ship with a lot of hit points.  The PSF and PF are the options for this.  A cloaked SF is sometimes also appropriate but that is circumstantial.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Dangerous routes are best handled by a ship with a lot of defenses (shields and armor). The PSF and PF are the best options for this.  A cloaked SF is sometimes also appropriate but that is circumstantial.
```

File: game-guide/trading.md [hunk 17]
```diff
@@ -1,54 +1,296 @@
...
+
+Long routes over about 13 in length are best traded with a jump ship.  The speed of an ACV helps more than you may think.  An IST has forces to help keep it in one piece.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Long routes over about 13 distance are best traded with a jump ship.  The speed of an ACV makes it the most efficient ship for this purpose; however, an IST has forces that help make trading a bit safer.
```

File: game-guide/trading.md [hunk 18]
```diff
@@ -1,54 +1,296 @@
...
+Alskant ships are the strongest traders in the game.  Every single one of their trade ships is the best in some way.  Here they are in the same format as the neutral traders:
+
+|Ship|Cargo|Turn Rate|Trade Power|Credits|Credits/T.P.|Total Health|Combat Drones|Scouts|Mines|S|C|I|J|S|
+|----|-----|---------|-----------|-------|------------|------------|-------------|------|-----|-|-|-|-|-|
+|Trip-Maker|235|12|2820|1354955|481|350|0|0|0|Y|N|Y|N|N|
+|Deep-Spacer|300|8|2400|5721889|7629|750|50|20|25|Y|N|N|Y|N|
+|Deal-Maker|200|11|2200|5727059|2603|540|30|10|15|Y|Y|N|N|N|
+|Trade-Master|200|9|1800|12095764|6720|1500|100|25|100|Y|N|N|N|Y|
+
+The trip maker is your mid tier trader.  It is the strongest trader in the game and a death trap with 350 total hit points.  Get in it quickly, trade until people start hunting, and move on to a larger ship before you get podded.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
The Trip-Maker is your mid tier trader.  It is the strongest trader in the game and a death trap with 350 total defenses.  Get in it quickly, trade until people start hunting, and move on to a larger ship before you get podded.
```

File: game-guide/trading.md [hunk 19]
```diff
@@ -1,54 +1,296 @@
...
+
+The deep spacer is an excellent general purpose trader as well as the top jump route trader in the game.  This is a common ship to see Alskant traders in because it's usually the best choice.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
The Deep-Spacer is an excellent general purpose trader as well as the top jump route trader in the game.  This is a common ship to see Alskant traders in because it's often the best choice.
```

File: game-guide/trading.md [hunk 20]
```diff
@@ -1,54 +1,296 @@
...
+
+The deal maker is the strongest cloaked trader in the game.  You will not see them often but they fill a fun niche when the game allows for it.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
The Deal-Maker is the strongest cloaked trader in the game.  You will not see them often but they fill a fun niche when the game allows for it.
```

File: game-guide/trading.md [hunk 21]
```diff
@@ -1,54 +1,296 @@
...
+
+The trade master is a misnomer.  It is a bad choice for trading.  It's true purpose is alliance support.  Those roles be covered in a different guide.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
The Trade-Master is a misnomer.  It is surprisingly a bad choice for trading.  It's true purpose is alliance support.  Those roles are covered in a different guide.
```

File: game-guide/trading.md [hunk 22]
```diff
@@ -1,54 +1,296 @@
...
+
+Never underestimate forces.  Scouts can provide valuable warning when a hunter enters your space.  Mines slow down anyone who hits them.  A single combat drone is able to absorb all 300 damage from a Nuke.
+
+A well configured illusion generator may confuse someone for just long enough to save your ship.  But hunters quickly learn to expect it.
+
+## Routes Length Is Important
+
+This can't be said enough.  Longer routes allows higher distance factors which helps everything about trading in a major way.
+
+Longer distance factors directly multipliers the number of credits like so:
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Trade profit increases very quickly with increasing distance factors:
```

File: game-guide/trading.md [hunk 23]
```diff
@@ -1,54 +1,296 @@
...
+
+|Distance Number|Multiplier|
```
Comment (hemberger) [inline game-guide/trading.md:221]:
```suggestion
|Distance Factor|Profit Multiplier|
```

File: game-guide/trading.md [hunk 24]
```diff
@@ -1,54 +1,296 @@
...
+|---------------|----------|
+|1|1|
+|2|2.46|
+|3|4.17|
+|4|6.06|
+|5|8.10|
+|6|10.27|
+
+The longer the route, the more each good is worth in terms of credits.
```
Comment (hemberger) [inline game-guide/trading.md:230]:
```suggestion
```

File: game-guide/trading.md [hunk 25]
```diff
@@ -1,54 +1,296 @@
...
+
+If experience is what you're after, here are some sample values per trade at each distance:
+
+|Route Length|100 Cargo XP|200 Cargo XP|300 Cargo XP|
```
Comment (hemberger) [inline game-guide/trading.md:234]:
```suggestion
|Distance Factor|100 Cargo XP|200 Cargo XP|300 Cargo XP|
```

File: game-guide/trading.md [hunk 26]
```diff
@@ -1,54 +1,296 @@
...
+|1|8|15|23
+|2|15|31|46
+|3|23|46|69
+|4|31|62|92
+|5|38|77|115
+|6|46|92|138
+
+## Dumping Goods
+
+Dumping of cargo can be used to convert experience in to credits.  This is done by manipulating port resupply.  Say you've run out of luxury goods to trade but there are still tier 2 goods in the port.  You can buy the goods and dump them until the luxury goods are resupplied enough to keep trading.
```
Comment (hemberger) [inline game-guide/trading.md:245]:
```suggestion
Dumping of cargo can be used to help resupply a port. Say you've run out of Luxury Items to trade but there are still tier 2 goods in the port.  You can buy the tier 2 goods and dump them until the Luxury Items are resupplied enough to keep trading. However, because you lose experience when dumping cargo, this strategy should only be used when your goal is to maximize profit, not experience.
```

File: game-guide/trading.md [hunk 27]
```diff
@@ -1,54 +1,296 @@
...
+
+The experience loss I mentioned is indirect.  The buy and loss action is a net 0 for experience but costs you two turns.  Further, the resupply rate is 90% so you need to buy/dump 1.11 times to resupply one trade.  Every buy/dump run effectively tacks on 2.22 turns to the run.  That makes the same route cost 37% more turns on a 1 sector route and a 13% more on a 6 sector.  You can force a lot of cash out of a route like this but it is often better to swap to another route.
```
Comment (hemberger) [inline game-guide/trading.md:247]:
```suggestion
```

File: game-guide/trading.md [hunk 28]
```diff
@@ -1,54 +1,296 @@
...
+
+The effective way to use dumping is to keep good supplies nearly topped off.  The reason is in the supply portion of the ideal price equation.  If supply is 0 that feeds a 1 multiplier into the equation.  If supply is full then it will be a 3.  Keeping a port full will get you three times the credits.  This can be significant if you're hard up for cash.
```
Comment (hemberger) [inline game-guide/trading.md:249]:
```suggestion
_Tip_: When selling goods, you make about 3x the profit when demand is high (due to the way supply/demand factors into the "ideal price" equation). Buying and dumping tier 2 goods to keep the demand of tier 3 goods high can sometimes be an effective way to maximize profit at the expense of some experience.
```

File: game-guide/trading.md [hunk 29]
```diff
@@ -1,54 +1,296 @@
...
+
+Stealing has more serious ramifications beyond being caught.  Where a normal trade would build relations stealing does not.  This, in combination with lost relations when you are caught, makes theft a terrible idea when relations are below 1000.  Further, stolen goods do not regenerate higher tier goods.  This can cause your routes to drain early and severely shorten profits.  These effects limit the scenarios where stealing is efficient in terms of profit.
+
+## Hazards
+
+Hunters are human.  That's one of the things that makes this game interesting.  The problem is this is a trading guide which means we are prey.  You are the prey and you will be hunted by people who learn and adopt new tactics as time goes on.  When caught off guard you will most likely lose your ship in a hail of weapons fire.
+
+One of the simplest ways hunters search for prey is watching the experiene of other players.  Experience is only gained from certain actions.  And trading is the only one that causes it to steadily trickle in.  The amounts of experience you gain give clues as to how large your ship is and where you may be trading.  It is nearly impossible to trade without others knowing it.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
One of the simplest ways hunters search for prey is watching the experience of other players.  Experience is only gained from certain actions, and trading has an obvious signature.  The amount of experience you gain give clues as to how large your ship is and where you may be trading.  It is nearly impossible to trade without others knowing it.
```

File: game-guide/trading.md [hunk 30]
```diff
@@ -1,54 +1,296 @@
...
+
+The current player list is fundamental tool for hunters and traders alike.  The count at the top is the number of players who have loaded or refreshed a page in the past 10 minutes.  The list of players are those who have moved in the same time period.  These are a good indicator of people being around but not perfectly accurate.  It is possible to leave the current player list open for much longer than 10 minutes.  You never know who is watching.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
The current player list is a fundamental tool for hunters and traders alike.  The count at the top is the number of players who have loaded or refreshed a page in the past 10 minutes.  The list of players are those who have moved in the same time period.  These are a good indicator of people being around but not perfectly accurate.  It is possible to leave the current player list open for much longer than 10 minutes.  You never know who is watching.
```

File: game-guide/trading.md [hunk 31]
```diff
@@ -1,54 +1,296 @@
...
+
+The current hall of fame is an absolute goldmine for espionage.  Covering it fully would require a separate guide.  For the scope of this section just be aware of it and take a look to get an idea of what is there.
+
+Trading often means dealing with forces.  Scouts send a message to their own when a ship passes through.  Combat drones provide armor and deal damage.  Mine deal damage and interrupt you with a combat screen when you enter a sector.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Trading often means dealing with forces.  Scouts send a message to their owner when a ship passes through.  Combat drones provide armor and deal damage.  Mines deal damage and interrupt you with a combat screen when you enter a sector.
```

File: game-guide/trading.md [hunk 32]
```diff
@@ -1,54 +1,296 @@
...
+
+Scouts can be the greatest ship saving tool for a trader.  Place them at the entrance to the galaxy where you're trading.  Place them in key places along your route.  You will get a message when someone is coming for you.  That's your air raid warning telling you to take cover.
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
Scouts can be the greatest ship saving tool for a trader.  Place them at the entrance to the galaxy where you're trading.  Place them in key sectors along your route.  You will get a message when someone is coming for you.  That's your air raid warning telling you to take cover.
```

File: game-guide/trading.md [hunk 33]
```diff
@@ -1,54 +1,296 @@
...
+
+The effective way to use dumping is to keep good supplies nearly topped off.  The reason is in the supply portion of the ideal price equation.  If supply is 0 that feeds a 1 multiplier into the equation.  If supply is full then it will be a 3.  Keeping a port full will get you three times the credits.  This can be significant if you're hard up for cash.
+
+## Smuggling
+
+All ships are checked for illegal goods at random and it is possible to avoid under the right circumstances.  A port checks 15% of ships for illegal goods by default.  It is possible to reduce the chance by 16%.  There is a 4% reduction for each illegal good, up to 3 max, a port trades.  There is also a 4% reduce in chance when flying an evil aligned ship (Thief, Assassin, Death Cruiser).
```
Comment (hemberger) [inline game-guide/trading.md]:
```suggestion
All ships are checked for illegal goods at random, but it is possible to reduce how frequently this occurs.  A port checks 15% of ships for illegal goods by default.  It is possible to reduce the chance by 16%.  There is a 4% reduction for each illegal good a port trades, up to 3 max.  There is also a 4% reduction when flying an evil aligned ship (Thief, Assassin, Death Cruiser).
```

**Review by hemberger — COMMENTED** [review]
These are my final suggested edits. Thanks for your work on this, it'll be a great guide for new players who really want to dig deep into the various aspects of trading!

**Comment by hemberger** [comment]
Thanks for the guide!
